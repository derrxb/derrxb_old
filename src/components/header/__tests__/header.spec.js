import React from 'react';
import renderer from 'react-test-renderer';
import Header from '../index';

describe('Header Spec', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<Header nature="default" />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
