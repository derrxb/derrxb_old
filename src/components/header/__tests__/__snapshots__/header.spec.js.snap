// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header Spec renders correctly 1`] = `
<nav
  className="sc-EHOje jQwEIl"
>
  <a
    className="sc-bZQynM lnZZkT"
    href="/"
  >
    
  </a>
  <button
    className="sc-htpNat PzwZx"
    onClick={[Function]}
  >
    Menu
  </button>
  <div
    className="sc-bdVaJa iORtBj"
  >
    <a
      className="sc-bwzfXH dVvytM"
      href="/"
    >
      Home
    </a>
    <a
      className="sc-bwzfXH dVvytM"
      href="/blog"
    >
      Blog
    </a>
    <a
      className="sc-bwzfXH dVvytM"
      href="/photography"
    >
      Photography
    </a>
    <a
      className="sc-bwzfXH dVvytM"
      href="/book"
    >
      Book Session
    </a>
    <a
      className="sc-bwzfXH dVvytM"
      href="/about"
    >
      About
    </a>
  </div>
</nav>
`;
